src/libraries/Microsoft.PowerFx.Core/Types/DType.cs

```diff
@@ -1528,86 +1510,19 @@ public DType DropMulti(ref bool fError, DPath path, params DName[] rgname)
                 fError = true;
                 return this;
             }
-
-            Contracts.Assert(typeOuter.IsRecord || typeOuter.IsTable);
-
-            var tree = typeOuter.TypeTree.RemoveItems(ref fError, rgname);
-
-            return SetType(ref fError, path, new DType(typeOuter.Kind, tree, AssociatedDataSources, DisplayNameProvider));
-        }
-
-        // Drop everything but the specified names/fields from path's type, and return the resulting type.
-        // If a name/field (that was specified to be kept) is missing, we are returning a new type
-        // with the type for the missing field as Error and fError will be true.
-        public DType KeepMulti(ref bool fError, DPath path, params DName[] rgname)
```

Comment:
>KeepMulti [](http://example.com/codeflow?start=21&length=9)

Why are we removing keepMulti and Split? Are they not used / bad patterns?

Reply:
Yes, unused. I left a comment above about  this. Pulling this cleanup to a separate PR